lib/liquid/standardfilters.rb

```diff
@@ -394,6 +416,10 @@ def default(input, default_value = ''.freeze)
 
     private
 
+    def raise_property_error(property)
+      raise Liquid::ArgumentError.new("Liquid error: cannot select the property '#{property}'")
```

Comment:
What @pushrax meant is that the `Liquid error:` prefix is added automatically further up the stack inside Shopify. This should be:
```ruby
raise Liquid::ArgumentError.new("cannot select the property '#{property}'")
```

Reply:
Ah good catch...